### DISCUSSION TITLE
Donations?

### DISCUSSION THREAD

**Discussion by kclax** [tenacityteam/tenacity-legacy#139] [Q&A] [answered]
I am not a coder but would love to help the project if possible? Not sure if donations would at this point?

**Comment by l0go** [comment]
As of this moment I cannot think of much which will need to be donated for. The website is pretty much the only thing that costs money and that is only a few bucks for a domain and fosshost or similar can host.

**Comment by fossdd** [comment]
See https://github.com/tenacityteam/tenacity/issues/67. In short: Currently there is not a donate method. But it's awesome to see that people want to donate to this project.

**Comment by JYamihud** [comment]
We need to open official pages in all ( Free Software ) places where you can donate money.
 - [LibrePay](https://liberapay.com/)
 - [Odysee (LBRY)](https://odysee.com/)

Suggest more

**Reply by SubOptimal** [reply]
Would it be also an option to use GitHub Sponsors
https://docs.github.com/en/sponsors/getting-started-with-github-sponsors/about-github-sponsors

**Reply by Semisol** [reply]
> Would it be also an option to use GitHub Sponsors
> [docs.github.com/en/sponsors/getting-started-with-github-sponsors/about-github-sponsors](https://docs.github.com/en/sponsors/getting-started-with-github-sponsors/about-github-sponsors)

proprietary

**Reply by fossdd** [reply]
> * [LibrePay](https://liberapay.com/)

Is nice, of course. And it's open source, but it uses Cloudflare, which is properitary and they aren't looking to switch.

> * [Odysee (LBRY)](https://odysee.com/)

That's more for video, etc. And it's yet another crypto currency

**Comment by emabrey** [comment, marked as answer]
I have setup an Open Source Collective based donation option. You can donate using the Github Sponsors feature and the money will be placed into the control of the Fiscal Host, which is the Open Source Collective. All expenditures will be public and require approval, that way donators know that their donations are being used appropriately.

The Open Source Collective for Tenacity is located [here](https://opencollective.com/tenacity).
Donations can be submitted there directly, or alternatively, you can [use the GitHub sponsors feature](https://github.com/sponsors/tenacityteam).

Some of our developers are also independently offering donation options that DO NOT go to the project, but rather directly to the individual, myself included. I want to clarify that if you donate via our individual profile pages ([mine](https://github.com/sponsors/emabrey), for example) that DOES NOT GO TO THE PROJECT. There is nothing untoward about offering individual donation options, of course, but I do not want there to be any confusion.
